doc/getting-started/install.md

```diff
@@ -90,7 +90,7 @@ Create a working directory for your builds:
     ./configure
     sudo make install
 
-This assumes GHC 8.10.2 on Linux (the most recent version at the time of writing).  If you are installing on MacOSX or Windows, download the compiler from `https://www.haskell.org/platform/mac.html` instead, and follow the installation instructions.
```

Comment:
This should work with `ghc-8.10.3` as well since we compile with that in Github Actions.

Reply:
Hmm, I made a mistake.  We don't actually for this project.